src/test/java/crystal/command/DeadlineCommandTest.java

```diff
@@ -0,0 +1,21 @@
+package crystal.command;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Represents the DeadlineCommandTest class.
+ *
+ */
+public class DeadlineCommandTest {
```

Comment:
Sufficient test cases are provided for every task objects classes !! :+1::thumbsup: